inst/shiny/qcoder/app.R

```diff
@@ -254,22 +269,17 @@ if (interactive()) {
       output$units_table <- DT::renderDataTable({
         if (units_df_path == "") {return()}
         units_df <- readRDS(units_df_path)
-        DT::datatable(units_df,
-                      options = list(paging = FALSE, dom = "Bfrtip",
```

Comment:
Same here ... please update your branch to the master branch because you would be deleting a lot of the changed code.